### PR TITLE
MAUI-871351-DocUGStepProgressBar-Content added.

### PR DESCRIPTION
**Pull request by IndumathiR-1995**
Hi @MuniappanSubramanian ,

I have added the UG content for the below topics.

- Events
- Accessibility
- Customization

Regards,
Indumathi R

### CODE REVIEW

File: MAUI/StepProgressBar/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<tr>
+<td>Step</td>
+<td>Step Number PrimaryText/SecondaryText Stepstatus</td>
+<td>Step 1 Login Completed </td>
+<td>Step</td>
+<td>Step Number PrimaryFormattedText/SecondaryFormattedText Stepstatus</td>
+<td>Step 1 Login Completed </td>
+<td>Step</td>
+<td>Step Number PrimaryText+SecondaryText Stepstatus</td>
+<td>Step 1 Login date Completed </td>
```
Comment (MuniappanSubramanian) [inline MAUI/StepProgressBar/accessibility.md:29]: Check table alignment

File: MAUI/StepProgressBar/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+<td>Step</td>
+<td>Step Number PrimaryFormattedText+SecondaryFormattedText Stepstatus</td>
```
Comment (MuniappanSubramanian) [inline MAUI/StepProgressBar/accessibility.md:31]: Update proper example

File: MAUI/StepProgressBar/customization.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+By using the [ProgressBarBackground](), [ActiveStepProgressValue](), [ActiveStepIndex]() you can customize the step progress bar.
+
+{% tabs %}
+{% highlight xaml hl_lines="1" %}
+
+<ContentPage.BindingContext>
+    <local:ViewModel/>
+</ContentPage.BindingContext>
+
```
Comment (MuniappanSubramanian) [inline MAUI/StepProgressBar/customization.md:81]: No need to add viewmodel for all cases

File: MAUI/StepProgressBar/customization.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+sfstepProgressbar.CompletedStepSettings = completedStepSettings;
+
+this.Content = sfstepProgressbar;
+
+{% endhighlight %}
+{% endtabs %}
+
+## ProgressBar customization
+
+By using the [ProgressBarBackground](), [ActiveStepProgressValue](), [ActiveStepIndex]() you can customize the step progress bar.
```
Comment (MuniappanSubramanian) [inline MAUI/StepProgressBar/customization.md:73]: Seems like simply explained- Explain in detail

File: MAUI/StepProgressBar/customization.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+description: Learns about Customization support with Syncfusion .NET MAUI Step ProgressBar (SfStepProgressBar)
+platform: maui
+control: SfStepProgressBar
+documentation: ug
+---
+
+# Customization in MAUI StepProgressBar
+
+## Step content customization
+You can customize the step [ContentType]() with [Numbering](), [Tick](), [Cross](), [Dot]() and [Images]().
```
Comment (MuniappanSubramanian) [inline MAUI/StepProgressBar/customization.md:13]: Image customization?
Formatted Tex customization?

File: MAUI/StepProgressBar/events.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+{
+// Handle event action
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## StepChanged
+
+The [`StepStatusChanged`]() event is triggered when changing the step view status. The associated argument contains the following information.
```
Comment (MuniappanSubramanian) [inline MAUI/StepProgressBar/events.md:40]: Changing- Check

File: maui-toc.html [hunk 7]
```diff
@@ -1045,6 +1045,9 @@
             <li><a href="/maui/StepProgressBar/overview">Overview</a></li>
             <li><a href="/maui/StepProgressBar/getting-started">Getting Started</a></li>
             <li><a href="/maui/StepProgressBar/migration">Migrate from Xamarin.Forms</a></li>
+			<li><a href="/maui/StepProgressBar/customization">Customization</a></li>
+			<li><a href="/maui/StepProgressBar/events">Events</a></li>
+			<li><a href="/maui/StepProgressBar/accessibility">Accessibility</a></li>
```
Comment (MuniappanSubramanian) [inline maui-toc.html:1050]: Events 
Customization
Accessibility

File: MAUI/StepProgressBar/accessibility.md (hunk 1 above)
Comment (IndumathiR-1995) [reply MAUI/StepProgressBar/accessibility.md:29]: Table format aligned

File: MAUI/StepProgressBar/events.md (hunk 6 above)
Comment (IndumathiR-1995) [reply MAUI/StepProgressBar/events.md:40]: Description updated with **Changing**

File: maui-toc.html (hunk 7 above)
Comment (IndumathiR-1995) [reply maui-toc.html:1050]: Order updated in toc file

File: MAUI/StepProgressBar/customization.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+description: Learns about Customization support with Syncfusion .NET MAUI Step ProgressBar (SfStepProgressBar)
+platform: maui
+control: SfStepProgressBar
+documentation: ug
+---
+
+# Customization in MAUI StepProgressBar
+
+## Step content customization
+You can customize the step content type, size, active step index, strole width and acctive step progress value by using these properties, [ContentType]() with [Numbering](), [Tick](), [Cross](), [Dot]() and [Images](), [StepSize](), [StepContentSize](), [ActiveStepIndex](), [StepStrokeWidth](), [ActiveStepProgressValue]().
```
Comment (MuniappanSubramanian) [inline MAUI/StepProgressBar/customization.md:13]: acctive - spell error

**Pull request closed without merging**